projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1331,6 +1331,33 @@ public static function site_supports_next_default_size() {
 		// Final fallback.
 		return false;
 	}
+
+	/**
+	 * Register block metadata collection for Jetpack blocks.
+	 * This allows for more efficient block metadata loading by avoiding
+	 * individual block.json file reads at runtime.
+	 *
+	 * Uses wp_register_block_metadata_collection() if available (WordPress 6.7+)
+	 * and if the manifest file exists. The manifest file is auto-generated
+	 * during the build process.
+	 *
+	 * Runs on plugins_loaded to ensure registration happens before individual
+	 * blocks register themselves on init.
+	 *
+	 * @static
+	 * @since $$next-version$$
+	 * @return void
+	 */
+	public static function register_block_metadata_collection() {
+		$meta_file_path = JETPACK__PLUGIN_DIR . 'extensions/blocks/blocks-manifest.php';
```

Comment:
Then this will need to change to match.
```suggestion
		$meta_file_path = JETPACK__PLUGIN_DIR . '_inc/blocks/blocks-manifest.php';
```

Reply:
Thanks, done in 17ae56bdcfb2